workspaces/vscode/install.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Apt dependencies
+apt-get update
+apt-get upgrade --yes
+apt-get install --yes python3-pip gosu
+
+# Install Python dependencies
+pip install scipy numpy matplotlib pandas
```

Comment:
I'd like to get thoughts from other folks, but IMO we shouldn't be opinionated about what software we install in this base image. If someone wants a workspace with a specific set of dependencies, they should create their own image with `prairielearn/workspace-vscode` as a base and install any deps they need. Otherwise, this image will just grow without bounds forever as people shoehorn more and more languages and libraries into it (or maybe that's all FUD 🤷).

Reply:
I think we should have one image with the basic python libraries, such as the ones above. I can imagine most people using python will want to use them. And if they want to get more libraries, yes, they can create their own image. 

I guess this is my selfish opinion. This request came because I am setting up a course for Brian (MechSE) for Finite Element Analysis, and I needed a workspace with these basic functions.

Reply:
I guess my issue is that this isn't a Python workspace, it's a generic VSCode workspace. If we want an opinionated Python+VSCode workspace, maybe we can do that ourselves by introducing a separate `vscode-python` image that builds on top of this generic `vscode` one?

Reply:
Sure, totally on-board with having a separate Python-based vscode image.

Reply:
Continuing the conversation, is there anything we should consider essential to a generic vscode workspace?

Reply:
IMO, the base image should contain just the stuff we need to get VSCode in the browser. Anything language-specific should be delegated to sub images, whether maintained by us or by individual courses.

That said, that's just _my_ opinion - let me know if that's too strict or impractical 😅